### PR TITLE
deps: k8s.io/kubernetes: use v1.3.0-alpha.4

### PR DESCRIPTION
**Pull request by steveeJ**
/cc @jonboulle

### CODE REVIEW

File: Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/resource/quantity.go [hunk 1]
```diff
@@ -88,34 +86,19 @@ import (
...
 // +protobuf.options.marshal=false
 // +protobuf.options.(gogoproto.goproto_stringer)=false
 type Quantity struct {
-	// i is the quantity in int64 scaled form, if d.Dec == nil
-	i int64Amount
-	// d is the quantity in inf.Dec form if d.Dec != nil
-	d infDecAmount
-	// s is the generated value of this quantity to avoid recalculation
-	s []byte
+	// Amount is public, so you can manipulate it if the accessor
```
Comment (jonboulle) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/resource/quantity.go]: ruh!?!? was this a reversion upstream? see https://github.com/appc/spec/pull/607/files#diff-6c84e496be2b46a45cb5c0b30224f6ebL17

**Comment by jonboulle** [comment]
It's extremely unclear to me why but apparently `v1.3.0-alpha.3-1204-g9625926` (from #607) is ahead of `v1.3.0-alpha.4` - even though if I check out the relevant commit myself (9625926852e215caa35e278c2cd7926744532291) it doesn't describe that way:

```
: trevize kubernetes master ; g log  v1.3.0-alpha.4..9625926852e215caa35e278c2cd7926744532291 -- pkg/api/resource/quantity.go
commit b2a01d4d94090b5efd81d0b71cb21bbb932c053a
Author: Clayton Coleman <ccoleman@redhat.com>
Date:   Tue May 17 00:35:40 2016 -0400

    Add an int64 quantity implementation
    
    Provides fast transformations of int64 scaled values with overflow to
    inf.Dec as needed. Attempts to reduce allocations and expensive
    operations in the critical path.
    
    Alters the public signature of Quantity to encourage more efficient use
: trevize kubernetes master ; g co 962592685
Note: checking out '962592685'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD ist jetzt bei 9625926852e2... Merge pull request #25574 from Clarifai/elb
: trevize kubernetes HEAD ; g describe --dirty
v1.3.0-alpha.4-205-g9625926852e2
```

**Comment by jonboulle** [comment]
I can only suppose that the Godep process in #607 went wrong somehow /cc @dlorenc

**Comment by jonboulle** [comment]
@steveeJ can you remove https://github.com/appc/spec/pull/614/commits/29f1228ead41e573c66c0c743861277c27b0d0a0 (i.e. un revert)

**Comment by alban** [comment]
> It's extremely unclear to me why but apparently v1.3.0-alpha.3-1204-g9625926 (from #607) is ahead of v1.3.0-alpha.4

It can happen if the person running godep didn't fetch the new tags (`git fetch --tags`) so godep only see that it is 1024 commits after v1.3.0-alpha.3 but don't see the remote tag v1.3.0-alpha.4.

@steveeJ could this be the case here?

**Comment by jonboulle** [comment]
ah! great spot

**Comment by jonboulle** [comment]
@steveeJ rebase please

**Comment by jonboulle** [comment]
and you'll need to drop ef295a2

**Comment by jonboulle** [comment]
outstanding

**Pull request merged**